developer-extension/README.md

```diff
@@ -40,3 +40,58 @@ Then, in Google Chrome:
 ## Browser compatibility
 
 For now, only Google Chrome is supported.
+
+## Usage Tips
+
+### Event Tab
+
+The Event Tab contains a list of events sent by the SDK and a menu of event types for quick filtering.
+
+#### Search Syntax
+
+We support a basic `key:value` search syntax which means you can search within the limits of [RUM event structures](https://docs.datadoghq.com/real_user_monitoring/explorer/search/), such as `action.target.name:action_name`. While all the fields can be added as columns in the event list, it does not mean the existing ones are valid fields: the `Description` summary is not part of RUM event structure. You can click on any `Description` cell for details of the event to look for correct key to search with.
+
+We split each search key based on whitespaces. To search with multiple conditions, simply add whitespaces in between, such as:
+
+```
+type:view application.id:2 action.target.name:my_action_name
+```
+
+#### Event Columns
+
+The Events List offers an interactive experience to visualize RUM events:
+
+- Drag and drop to reorder columns in the event list
+- Remove (by clicking on `x` in the column title) or add new columns:
+  - Add a new column from searching for a field by clicking on the `+column` icon at the right side of the header row.
+  - Add a new column from values in existing columns by right clicking on any attribute in the event json.
+- Copy queries and objects from the list by right clicking on any cell
+
+### Info Tab
+
+**⚠️Don’t forget to reset everything in the Info Tab after experimenting.**
+
+Info tab contains information about Session and RUM SDK configurations
+
+- **RUM/LOGS Configuration**: edit configuration files on the fly. When configuration changes apply, the extension will automatically reload the page. But for some configurations you might want to click on End Current Session to ensure that the changes kicked in.
+- **End current session**: manually end the current session within the extension. This will also end the current replay session.
+
+### Setting Tab
+
+**⚠️Don’t forget to reset everything in the Setting Tab after experimenting.**
```

Comment:
**💬 suggestion:** ‏Github flavored markdown has a special syntax for emphasis block: 
```suggestion
> [!IMPORTANT]  
> Don’t forget to reset everything in the Setting Tab after experimenting.
```
Renders as:
> [!IMPORTANT]  
> Don’t forget to reset everything in the Setting Tab after experimenting.

See: https://github.com/orgs/community/discussions/16925